src/babylon.node.ts

```diff
@@ -110,6 +110,9 @@
         /** @hidden */
         public _worldMatrixDeterminant = 0;        
 
+        /** @hidden */
+        private _sceneRootNodesIndex = -1;
+

```

Comment:
Note that if the order of children of a node does not matter, we could also keep the `_parentChildNodeIdx`to get a O(1) time removal from parent children array

Reply:
Yes agree